ucm2/NXP/iMX8/Librem_5/HiFi.conf

```diff
@@ -0,0 +1,180 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='Digital Playback Volume' 100,100"
+		cset "name='MIXINL IN2L Switch' off"
+		cset "name='MIXINL IN3L Switch' off"
+		cset "name='MIXINR IN2R Switch' off"
+		cset "name='MIXINR IN3R Switch' off"
+		cset "name='INPGAR IN1R Switch' off"
+		cset "name='INPGAR IN2R Switch' off"
+		cset "name='INPGAR IN3R Switch' on"
+		cset "name='INPGAR IN4R Switch' off"
+		cset "name='INPGAL IN1L Switch' off"
+		cset "name='INPGAL IN2L Switch' off"
+		cset "name='INPGAL IN3L Switch' off"
+		cset "name='INPGAL IN4L Switch' off"
+		cset "name='Input Mixer Switch' off,on"
+		cset "name='SPKOUTL PGA' Mixer"
+		cset "name='SPKOUTR PGA' Mixer"
+	]
+
+	DisableSequence [
+		cset "name='Digital Playback Volume' 0,0"
+	]
+
+	Value {
+	}
+}
+
+SectionDevice."Handset" {
+	Comment "Handset"
+
+	ConflictingDevice [
+		"Headphones"
+		"Speaker"
+	]
+
+	EnableSequence [
+		cset "name='Speaker Switch' off,on"
+		cset "name='DAC L/R Swap Switch' on"
+		cset "name='DAC Monomix Switch' on"
+		cset "name='SPKOUTR PGA' DAC"
+		cset "name='Speaker Volume' 121,121"
+		cset "name='Speaker Boost Volume' 2"
+	]
+
+	DisableSequence [
+		cset "name='Speaker Switch' off,off"
+		cset "name='DAC L/R Swap Switch' off"
+		cset "name='DAC Monomix Switch' off"
+		cset "name='SPKOUTR PGA' Mixer"
+	]
+
+	Value {
+		PlaybackChannels "2"
+		PlaybackPriority "100"
+		PlaybackVolume "name='Speaker Volume'"
+		PlaybackSwitch "name='Speaker Switch'"
```

Comment:
Missing 'PlaybackMixerElem "Speaker"` value for PA/PW to enable hardware volume control.